### PR TITLE
Branch

### PR DESCRIPTION
**Pull request by buhanec**

### CODE REVIEW

File: scripts/security/delete.sh [hunk 1]
```diff
@@ -2,36 +2,21 @@
...
+for PROGRAM in elasticsearch logstash kibana; do
+    if [[ $(sudo /usr/share/$PROGRAM/bin/$PROGRAM-plugin list | grep -s -i x-pack) ]]; then
+        if [[ $(ps -p 1 | grep systemd) ]]; then
+            sudo systemctl stop $PROGRAM
+        elif [[ $(ps -p 1 | grep init) ]]; then
+            sudo service $PROGRAM stop
+        fi
+
+        sudo /usr/share/$PROGRAM/bin/$PROGRAM-plugin remove x-pack
+        sudo rm -rf /etc/$PROGRAM/x-pack
```
Comment (piotrbrzo) [inline scripts/security/delete.sh]: This folder exists only for elasticsearch - not kibana or logstash

Comment (buhanec) [inline scripts/security/delete.sh]: not an issue because -f

Comment (buhanec) [inline scripts/security/delete.sh]: why are we discussing this here

Comment (piotrbrzo) [inline scripts/security/delete.sh]: Because you opened a pull requst - which are for that explicit reason.

**Pull request merged**